drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -53,14 +53,57 @@ def record_forms
       end
     end
 
-    def service_forms
-      @service_forms ||= begin
+    def client_override(file_path)
+      return file_path unless ENV['CLIENT'].present?
+
+      client_override_fpath = file_path.gsub('/default/', "/#{ENV['CLIENT']}/")
+      if File.exist?(client_override_fpath)
+        client_override_fpath
+      else
+        file_path
+      end
+    end
+
+    # { ROLE => { identifier => definition }}
+    def record_forms_by_role
```

Comment:
It sure feels like all of this infrastructure is getting close to being generic enough it would be fun to publish as a gem or something.  Custom form implementation for GQL....